bin/flutter

```diff
@@ -73,6 +73,7 @@ function upgrade_flutter () {
 
   local revision=`(cd "$FLUTTER_ROOT"; git rev-parse HEAD)`
   if [ ! -f "$SNAPSHOT_PATH" ] || [ ! -s "$STAMP_PATH" ] || [ `cat "$STAMP_PATH"` != "$revision" ] || [ "$FLUTTER_TOOLS_DIR/pubspec.yaml" -nt "$FLUTTER_TOOLS_DIR/pubspec.lock" ]; then
+    rm -f "$FLUTTER_ROOT/version"
```

Comment:
Does this need to be mirrored in the Windows version of the script (`flutter.bat`)?

Reply:
Added:
```bat
    IF EXIST "version" DEL version
```
...just after `:do_snapshot`.

Reply:
You probably want the argument to DEL in quotes as well, just in case there are spaces in FLUTTER_ROOT.